src/FirePokemon.java

```diff
@@ -0,0 +1,56 @@
+public class FirePokemon extends PokemonGeneral {
+    int temperatureAttack;
+    String food;
+    int amountOfBalls;
+
+    public FirePokemon(String name, int level, int hp, int temperatureAttack, String food, int amountOfBalls) {
+        super(name, level, hp);
+        this.temperatureAttack = temperatureAttack;
+        this.food = food;
+        this.amountOfBalls = amountOfBalls;
+    }
+
+
+    @Override
+    public void sound() {
+        System.out.println("Ffoehhh, Ffoehhh is mijn geluid");
+    }
+
+    public void flameThrower(int temperatureAttack) {
+        System.out.println( " gebruikt de flamethrower met een temperatuur van " + temperatureAttack);
+    }
+
+    public void pyroBall( ) {
```

Comment:
Heel goed dat je logica in je methodes zet. Zo komen je kalsse variablen ook echt tot hun recht. Ze hebben echt nut